### PR TITLE
Update Position docs and Apply Refactoring

### PR DESCRIPTION
**Pull request by notJoon**
# Description

- Update docs for public functions
- Refactored functions that were detected by complexity checks.

### CODE REVIEW

**Review by dongwon8247 — DISMISSED** [review]

File: position/_RPC_dry.gno [hunk 1]
```diff
@@ -7,7 +7,16 @@ import (
...
+// DryMint simulates a mint and returns the amount0, amount1 that would be minted.
+//
+// It calculates the square root ratios at these ticks
+// and determines the liquidity for the given amounts. Based on the current tick,
+// it calculates and returns the amounts of the two assets that would be minted.
+//
+// Returns:
+//   Two strings representing the amounts of the first and second assets
+//   that would be minted.
+// TODO: Write test
```
Comment (r3v4s) [inline position/_RPC_dry.gno:19]: write test

File: position/_RPC_dry.gno [hunk 2]
```diff
@@ -33,38 +42,34 @@ func DryMint(
...
-				sqrtLowerX96,
-				sqrtUpperX96,
-				i256Liquidity,
-			)
-			amount1 = i256.MustFromDecimal(amount1Str)
-		}
+	var amount0, amount1 *i256.Int
+	switch {
+	case liquidity.IsZero():
+		amount0 = i256.Zero()
```
Comment (r3v4s) [inline position/_RPC_dry.gno:60]: 73 ~ 74 line initialize nil to zero, therefor we can remove `case liquidity.IsZero()` logic

File: position/position.gno [hunk 3]
```diff
@@ -165,24 +178,26 @@ func mint(params MintParams) (uint64, *u256.Uint, *u256.Uint, *u256.Uint) {
...
+		u256.Zero(),
+		u256.Zero(),
+		false,
+	)
 	positions[tokenId] = position
 
 	return tokenId, liquidity, amount0, amount1
 }
 
+// -------------------------------------------------------------------------------------------- //
```
Comment (r3v4s) [inline position/position.gno:199]: what is purpose of this line ??

File: position/position.gno [hunk 4]
```diff
@@ -269,17 +280,16 @@ func increaseLiquidity(params IncreaseLiquidityParams) (uint64, *u256.Uint, *u25
...
-		AddLiquidityParams{
-			poolKey:        position.poolKey,
-			tickLower:      position.tickLower,
-			tickUpper:      position.tickUpper,
-			amount0Desired: params.amount0Desired,
-			amount1Desired: params.amount1Desired,
-			amount0Min:     params.amount0Min,
-			amount1Min:     params.amount1Min,
-		},
+	ll := newAddLiquidtyParams(
```
Comment (r3v4s) [inline position/position.gno:283]: `ll` seems like bad naming

**Review by r3v4s — CHANGES_REQUESTED** [review]
Write test cases, seems like you're refactoring crucial part of position contract

File: position/position.gno (hunk 3 above)
Comment (notJoon) [reply position/position.gno:199]: Just for separate the function. forgot to remove it

File: position/_RPC_dry.gno (hunk 1 above)
Comment (notJoon) [reply position/_RPC_dry.gno]: Added. `__TEST_RPC_dry_test.gno`

https://github.com/gnoswap-labs/gnoswap/pull/296/commits/299d6f8bd76599de0ad674ee87784fc7ccfe8f9f

File: position/_RPC_dry.gno (hunk 2 above)
Comment (notJoon) [reply position/_RPC_dry.gno]: https://github.com/gnoswap-labs/gnoswap/pull/296/commits/299d6f8bd76599de0ad674ee87784fc7ccfe8f9f

File: position/position.gno (hunk 4 above)
Comment (notJoon) [reply position/position.gno]: changed to `liqParams`

https://github.com/gnoswap-labs/gnoswap/pull/296/commits/299d6f8bd76599de0ad674ee87784fc7ccfe8f9f

File: position/position.gno (hunk 3 above)
Comment (notJoon) [reply position/position.gno]: Just for separate public function regions. removed https://github.com/gnoswap-labs/gnoswap/pull/296/commits/bd068e18e93ce491b97de030db6467e790f495f4

**Review by r3v4s — APPROVED** [review]

**Review by dongwon8247 — APPROVED** [review]

**Pull request merged**